checklist-service/sls-resources.yml

```diff
@@ -1,3 +1,14 @@
+cognitoAuthorizer:
```

Comment:
The new api-service will need to be documented in the README and added to the diagram :)